src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh

```diff
@@ -23,3 +22,4 @@ cat /tmp/toolstate/toolstates.json
 python3 "$X_PY" test --stage 2 check-tools
 python3 "$X_PY" test --stage 2 src/tools/clippy
 python3 "$X_PY" test --stage 2 src/tools/rustfmt
+python3 "$X_PY" test --stage 2 src/tools/miri
```

Comment:
Is it possible that this only gave us back test coverage on Linux, but not on Windows? That's a problem...